AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/SetupActivity.java

```diff
@@ -71,7 +71,8 @@
     private VideoStreamRenderer rendererView;
     private VideoStreamRendererView previewVideo;
     private Button setupMissingButton;
-    private ImageButton switchCameraButton;
+    private Button switchCameraButton;
+    private ImageButton switchCameraImage;
```

Comment:
Instead of two overlapping buttons, could we have a single ImageButton or ImageView with padding around the icon?